src/components/configuration/sdk-config-docs/ionic.js

```diff
@@ -106,6 +106,25 @@ const framework = docsPrefix => {
             ]
           }
         ]
+      },
+      security: {
+        serviceLogoUrl: '/img/security.svg',
+        serviceName: 'Security',
+        serviceDescription:
+          'Installs a security service which gives developers the ability to control their apps from the Mobile Security Console.',
+        setupText: 'Security SDK setup',
+        docsLink: `${docsPrefix}/security.html`,
```

Comment:
as above

Reply:
@finp We are going to address your concerns with the `docsLink` string in a separate PR. It affects more than just the MSS